CHANGELOG.md

```diff
@@ -96,6 +96,27 @@ We're planning to make the private `Phoenix.Component.MacroComponent` API that w
 
 Note: Colocated hooks require Phoenix 1.8+.
 
+> [!NOTE]
```

Comment:
@josevalim should ExDoc support this syntax as well? It's a bummer that we either use ExDoc admonition blocks and it renders nicely in the docs, or the GitHub syntax and it renders nicely on GitHub.

<img width="878" height="600" alt="image" src="https://github.com/user-attachments/assets/badf60dc-6307-43f6-9aaa-48849dc32fea" />

Reply:
No, I am also strongly against the GitHub syntax, because it introduces English notation to Markdown which is otherwise language agnostic. 🤷‍♂️

Reply:
Yes, I did not say we were better. 😂 

Also ours is a hack on top of IAL. If we were to actually extend markdown, then I would not include English keywords.